code/modules/clothing/suits/ego_gear/aleph.dm

```diff
@@ -191,12 +191,149 @@
+
+/obj/item/clothing/suit/armor/ego_gear/seasons
+	name = "season's greetings"
+	desc = "This is a placeholder."
+	icon_state = "spring"
+	armor = list(RED_DAMAGE = 60, WHITE_DAMAGE = 60, BLACK_DAMAGE = 60, PALE_DAMAGE = 60) // 240
+	attribute_requirements = list(
+							FORTITUDE_ATTRIBUTE = 80,
+							PRUDENCE_ATTRIBUTE = 100,
+							TEMPERANCE_ATTRIBUTE = 80,
+							JUSTICE_ATTRIBUTE = 80
+							)
+	var/current_season = "winter"
+	var/mob/current_holder
+	var/list/season_list = list(
+		"spring" = list("vernal equinox", "Some heavy armor, completely overtaken by foilage. \n This E.G.O. will transform to match the seasons."),
+		"summer" = list("summer solstice","It looks painful to wear! \n This E.G.O. will transform to match the seasons."),
+		"fall" = list("autumnal equinox","Even though it glows faintly, it is cold to the touch. \n This E.G.O. will transform to match the seasons."),
+		"winter" = list("winter solstice","In the past, folk believed that the dead would visit as winter came. \n This E.G.O. will transform to match the seasons.")
+		)
+
+/obj/item/clothing/suit/armor/ego_gear/seasons/Initialize()
+	. = ..()
+	AddElement(/datum/element/update_icon_updates_onmob)
+	ChangeSeasons()
+
+/obj/item/clothing/suit/armor/ego_gear/seasons/equipped(mob/living/carbon/human/user, slot)
+	. = ..()
+	if(!user)
+		return
+	current_holder = user
+
+/obj/item/clothing/suit/armor/ego_gear/seasons/dropped(mob/user)
+	. = ..()
+	current_holder = null
+
+/obj/item/clothing/suit/armor/ego_gear/seasons/proc/ChangeSeasons()
+	switch(current_season)
+		if("spring")
+			current_season = "summer"
+		if("summer")
+			current_season = "fall"
+		if("fall")
+			current_season = "winter"
+		if("winter")
+			current_season = "spring"
+	addtimer(CALLBACK(src, .proc/ChangeSeasons), 10 MINUTES) //this is hacky but it will work until the abnormality is released
+	Transform(current_holder)
+
+/obj/item/clothing/suit/armor/ego_gear/seasons/proc/Transform()
+	icon_state = "[current_season]"
+	update_icon_state()
+	if(current_holder)
+		current_holder.update_inv_wear_suit()
+		to_chat(current_holder,"<span class='notice'>[src] suddenly transforms!</span>")
+		playsound(current_holder, "sound/abnormalities/seasons/[current_season]_change.ogg", 50, FALSE)
+	name = season_list[current_season][1]
+	desc = season_list[current_season][2]
+	switch(current_season) //you are probably wondering why didn't I use setRating()... well, it did not work as intended. this is bad, really bad, but it works for now
+		if("spring")
+			armor = armor.modifyRating(white = 20, black = -20)
+			if(armor.pale > 60)
+				armor = armor.modifyRating(black = 20, pale = -20)
+		if("summer")
+			armor = armor.modifyRating(red = 20, white = -40, black = 20)
+		if("fall")
+			armor = armor.modifyRating(red = -40, white = 20, black = 20)
+		if("winter")
+			armor = armor.modifyRating(red = 20, black = -40, pale = 20)
```

Comment:
It's set up that way for a reason, and it's meant to be temporary until I set up an actual (small) subsystem for it linked to the abnormality.

Reply:
In the future this rotation could start at any point, and it will be tied to a global variable set by the abnormality (or the item in question, if it is currently null). While I have an idea of how to implement this already, I believe it will be easier to see the big picture when i'm working on the thing.